API/server.js

```diff
@@ -11,7 +11,7 @@ const sessionConfig = {
   name : "HelloSession" ,
   secret : "word" ,
   cookie : {
-      maxAge :  1000 * 20 , 
+      maxAge :  1000 * 50 , 
```

Comment:
You can set this to 50 seconds, but wouldn't you want the user to be logged in longer than that?